### PR TITLE
[KO] Renamed and applied enhancement for Modules Lesson

### PR DESCRIPTION
**Pull request by devleoper**
There was many changes in PR #424, but I thought that I must apply this first to prevent 404 Errors. There are also readability improvements.

@malkoG @Dalgona @riseshia Do you think this changes will make out translations more readable?

### CODE REVIEW

File: ko/lessons/basics/modules.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ iex> %{name: "Sean"} = sean
 
 ## 컴포지션
```
Comment (riseshia) [inline ko/lessons/basics/modules.md]: `컴포지션` -> `모듈`

**Comment by riseshia** [comment]
LGTM :)

**Comment by Dalgona** [comment]
:+1: :rocket:

**Comment by doomspork** [comment]
Thank you @devleoper, @riseshia, and @Dalgona !!

**Pull request merged**